### PR TITLE
docs: update READMEs for next-extraction-plugin & webpack-loader

### PR DESCRIPTION
**Pull request by layershifter**
This PR updates README files based on feedback from #190.

### CODE REVIEW

File: packages/next-extraction-plugin/README.md [hunk 1]
```diff
@@ -11,24 +10,47 @@ yarn add --dev @griffel/next-extraction-plugin
...
+      ],
+    });
+
+    // If your project uses TypeScript
+    config.module.rules.push({
+      test: /\.(ts|tsx)$/,
       exclude: /node_modules/,
       use: [
         {
           loader: '@griffel/webpack-loader',
```
Comment (yume-chan) [inline packages/next-extraction-plugin/README.md:49]: Also need `babelOptions` here.

Note that in a typical Next app, `@babel/preset-typescript` needs to be installed manually.

Maybe better to use `next/babel` preset which comes with Next, see https://nextjs.org/docs/advanced-features/customizing-babel-config

**Review by yume-chan — CHANGES_REQUESTED** [review]
I have tested my proposed changes with #195 patched locally. Both manually install `@babel/preset-typescript` and use `next/babel` directly works.

File: packages/next-extraction-plugin/README.md (hunk 1 above)
Comment (layershifter) [reply packages/next-extraction-plugin/README.md:49]: Good call, added 👍

File: packages/next-extraction-plugin/README.md [hunk 2]
```diff
@@ -11,24 +10,47 @@ yarn add --dev @griffel/next-extraction-plugin
...
 ## Usage
 
 In `next.config.js` file you'll need to add the next-plugin from `@griffel/webpack-extraction-plugin` like so:
+
 ```js
 // next.config.js
-const { withGriffelCSSExtraction } = require('@griffel/webpack-extraction-plugin');
+const { withGriffelCSSExtraction } = require('@griffel/next-extraction-plugin');
 
 module.exports = withGriffelCssExtraction()({
```
[the remark this answers is not in the document]
Comment (layershifter) [reply packages/next-extraction-plugin/README.md]: Oops, thanks 👍

**Review by yume-chan — APPROVED** [review]
LGTM.

File: packages/next-extraction-plugin/README.md [hunk 3]
```diff
@@ -11,24 +10,52 @@ yarn add --dev @griffel/next-extraction-plugin
 npm install --save-dev @griffel/next-extraction-plugin
 ```
 
+If you haven't installed yet `@griffel/webpack-loader`, please also install it:
```
Comment (ling1726) [inline packages/next-extraction-plugin/README.md:13]:
```suggestion
Please install `@griffel/webpack-loader` if you haven't done so already:
```

**Review by ling1726 — APPROVED** [review]

**Pull request merged**